.lintstagedrc.yml

```diff
@@ -1,4 +1,2 @@
 "*.js":
-  - "prettier-eslint --write"             # Run Prettier
```

Comment:
@j0k3r Does it mean it won't autofix from now on, but still prevents commit on lint errors?